### ISSUE TITLE
MissingFormatArgumentException when editing messages

### ISSUE THREAD

**Issue by morth1** [DV8FromTheWorld/JDA#362]
When I try to edit a message with a String that contains "%E" (comes up a lot in foreign URLs) I get a MissingFormatArgumentException.
For example, 
```
	String out = "%E";
	jda.getTextChannelById(303262002183274498L).editMessageById(messageID,out).queue();
```
Gives me the following exception:
```
Exception in thread "main" java.util.MissingFormatArgumentException: Format specifier '%E'
	at java.util.Formatter.format(Unknown Source)
	at java.util.Formatter.format(Unknown Source)
	at java.lang.String.format(Unknown Source)
	at net.dv8tion.jda.core.MessageBuilder.appendFormat(MessageBuilder.java:224)
	at net.dv8tion.jda.core.entities.MessageChannel.editMessageById(MessageChannel.java:1833)
	at Bot.main(Bot.java:32)
```

**Comment by MinnDevelopment** [comment]
This should get fixed by #361 
In the meantime you will need to use 2 % symbols in order to escape. like: `%%E` etc.

**Comment by MinnDevelopment** [comment]
This has been fixed with 2899a82
